### PR TITLE
Let node operators explore commit without trouble

### PR DESCRIPTION
**Pull request by NavyAdmiral**
Feeling like this adds positively to UX and personally like it more this way. 
Thoughts?

### CODE REVIEW

**Comment by dimroc** [comment]
@NavyAdmiral can you include a screenshot in frontend focused PRs please?

**Comment by NavyAdmiral** [comment]
@dimroc here's a gif
https://i.imgur.com/I4uJZy9.gif

**Comment by dimroc** [comment]
Should we change the link color so it’s discoverable?

**Comment by NavyAdmiral** [comment]
I don't think it should be getting in the eye and noisy. You're only supposed to discover it when you need it, and don't need to know about it if you don't need it

**Comment by NavyAdmiral** [comment]
That being said I'd be ok with a less noisy visual cue if you have something in mind

File: gui/src/components/Footer.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ const styles = theme => ({
     left: '0',
     bottom: '0',
     width: '100%'
+  },
+  bareAnchor: {
+    color: '#000000',
```
Comment (rupurt) [inline gui/src/components/Footer.js]: This color should come from the palette. `theme.palette.common.black`

**Pull request merged**